### ISSUE TITLE
Pass "cwd" option to rollups "createFilter" function

### ISSUE THREAD

**Issue by KingSora** [ezolenko/rollup-plugin-typescript2#250]
#### What happens and why it is wrong
I'm using `rollup-plugin-typescript2` to compile my project. But now I had a case where I needed to import files which are located not inside the directory where the `tsconfig.json` lies (also outside of the `process.cwd()` path).

I searched for quite a long time, until I figured out, that I have to change the `include` array of the plugin to get it working.
The change looks like: `[ "*.ts+(|x)", "**/*.ts+(|x)", "../**/*.ts+(|x)" ]`. I added the 3rd regex additional to the two which are there by default. (bit simplified here)

But then I thought, that those filters are matched agains the path provided by the `cwd` option, and just set this so I could keep the default includes. But it didn't work, so I digged a bit into the source and found that the solution could be to change this line: https://github.com/ezolenko/rollup-plugin-typescript2/blob/35b04d73ab9e0cb820bbf1f7637a714fc6974a18/src/get-options-overrides.ts#L74

into something like:
```js
return createRollupFilter(included, excluded, {
  resolve: pluginOptions.cwd
});
```
Because the `createFilter` function which rollup is generating is using the `process.cwd()` path by default, I thought it would make sense to also replace it with the `cwd` option of the plugin to have a consistend behavior.

I'm not sure if the proposed solution is good / bad, has any side-effects, or how to handle it properly etc.. I would really appreciate any guidance to my problem in case my solution is bs.

Thank you!

#### Environment
Windows 10
Node v12.17.0

##### Versions
```text
rollup: ^2.22.1 => 2.26.11
rollup-plugin-typescript2: ^0.27.1 => 0.27.2
typescript: ^3.9.7 => 3.9.7
```

**Comment by ezolenko** [comment]
I just merged a PR #249 from NicolasThierion with similar change using `compilerOptions.rootDir`. Will that work for your case?

**Comment by KingSora** [comment]
That solves it, thanks! :)
